kovri.md

```diff
@@ -46,7 +46,19 @@ Let's next go over some fairly technical stuff to have a better understanding of
 
 ## 5) Kovri and Technical Attributes
 
-> **TODO**: Give a concise overview of garlic routing/encryption
+Now hold on, since we are about to overview how Kovri works.
+
+Kovri is currently built from I2P's open standards. I2P stands for the invisible internet protocol. Kovri acts as an anonymizing router as it tunnels traffic through the I2P network.
+
+To protect their anonymity, each participant connects to other network members and uses them to pass messages.
+In most circumstances, people will choose four peers: two for incoming connections, and two for outgoing connections.
+People can use more for greater privacy at the cost of performance.
```

Comment:
This isn't a very accurate description of uni-directional tunnels and peer selection so I'm wondering if we should leave this out because it's well documented elsewhere.